app/config/index.js

```diff
@@ -79,10 +74,10 @@ export const getConfig = () => {
     cache: {
       expiresIn: THREE_DAYS_IN_MILLIS,
       options: {
-        host: process.env.REDIS_HOSTNAME || "redis-hostname.default",
+        host: process.env.REDIS_HOSTNAME ?? "redis-hostname.default",
         partition: "ffc-ahwr-frontend",
         password: process.env.REDIS_PASSWORD,
-        port: Number(process.env.REDIS_PORT) || DEFAULT_REDIS_PORT,
+        port: Number.parseInt(process.env.REDIS_PORT ?? "6379", 10),
```

Comment:
I see the sonarcloud flagging has been bypassed here for this being a magic number, because it has become a string. Wonder if it's worth just still having a const for the default port anyway? No biggie, but keeps inline with the default server port